ui/src/app/styles/colors.ts

```diff
@@ -15,7 +15,8 @@ export default {
     '#85bde4',
     '#afd3ed',
     '#d7eaf6',
-    '#eaf4fb'
+    '#eaf4fb',
+    '#2F2E7E'
```

Comment:
I really try to avoid changing the order of colors that are being referred from multiple files and has high chances to create issues which are difficult to catch. We can talk more about this offline